nav2_follow_path_controller/src/nav2_follow_path_controller.cpp

```diff
@@ -12,66 +12,80 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "dwb_controller/dwb_controller.hpp"
-
 #include <chrono>
 #include <memory>
 #include <string>
 
-#include "dwb_core/exceptions.hpp"
+#include "nav2_local_planner_interface/exceptions.hpp"
 #include "nav_2d_utils/conversions.hpp"
 #include "nav2_util/node_utils.hpp"
-#include "dwb_controller/progress_checker.hpp"
+#include "nav2_follow_path_controller/progress_checker.hpp"
+#include "nav2_follow_path_controller/nav2_follow_path_controller.hpp"
 
 using namespace std::chrono_literals;
 
-namespace dwb_controller
+namespace nav2_follow_path_controller
 {
 
-DwbController::DwbController()
-: LifecycleNode("dwb_controller", "", true)
+nav2_follow_path_controller::nav2_follow_path_controller()
+: LifecycleNode("nav2_follow_path_controller", "", true),
+  blp_loader_("nav2_local_planner_interface", "nav2_local_planner_interface::LocalPlannerInterface")
 {
-  RCLCPP_INFO(get_logger(), "Creating");
+  RCLCPP_INFO(get_logger(), "Creating local planner");
 
   declare_parameter("controller_frequency", 20.0);
 
-  // The costmap node is used in the implementation of the DWB controller
-  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>(
-    "local_costmap", nav2_util::add_namespaces(std::string{get_namespace()}, "local_costmap"));
+  // The costmap node is used in the implementation of the local planner
+  costmap_ros_ = std::make_shared<nav2_costmap_2d::Costmap2DROS>("local_costmap");
```

Comment:
The controller is constructed in the on_configure task, this should probably be there too

Reply:
I agree.

Reply:
The ```costmap_node``` created here is being passed to launch the ```costmap_thread``` during initialization, so didn't move it.